packages/pancake-uikit/src/theme/base.ts

```diff
@@ -5,7 +5,7 @@ export const breakpointMap: { [key: string]: number } = {
   sm: 576,
   md: 852,
   lg: 968,
```

Comment:
So we're leaving it at 968 for now? (as opposed to changing to 1080) 🙂

Reply:
Yea - there's wider use of that lg breakpoint as-is, so would like to chat with @hachiojidev & @ChefBugs to check we're all in agreement before making that change.